pipelines/tasks/clean-k8s.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env sh
+
+set -euo pipefail
+
+echo "Seting up bluemix access"
+ibmcloud logout
+ibmcloud login -a "$ibmcloud_server" --apikey "$ibmcloud_apikey"
+ibmcloud cs  region-set "$ibmcloud_region"
+
+echo "Running in cluster: $ibmcloud_cluster"
+
+export BLUEMIX_CS_TIMEOUT=500
+
+eval "$(ibmcloud cs cluster-config "$ibmcloud_cluster" --export)"
+
+OLD_DATE=$(date -d '4 hour ago' "+%s")
+export OLD_DATE
+
+delete_old() {
+  res="$1"
+
+  list=$(kubectl get "$res" --no-headers -o json | jq -r '
+    .items[] |
+    select(.metadata.creationTimestamp | fromdateiso8601 < ( env.OLD_DATE | tonumber ) ) |
+    select(.metadata.name | contains("test")) | .metadata.name
+  ')
+
+  if [ -z "$list" ]; then
+    echo "Currently no $res, older than 4 hours. Nothing to delete"
+  else
+    echo $list | xargs -r -n 50 kubectl delete --wait=false $res
```

Comment:
I did include a `--timeout=60s` before, while some namespaces shown that they required sometime to get fully delete, otherwise it timeout after 30s. Could we have this timeout again?

Reply:
@qu1queee  I checked the docs

```
--timeout=0s: The length of time to wait before giving up on a delete, zero means determine a
timeout from the size of the object
```

I was under the impression, that previous timeouts were mostly inserted due to finalizers and we don't run into that again.

But if infra is not fast enough to delete the namespace in the default time, I can add it back

Reply:
take a look on https://ci.flintstone.cf.cloud.ibm.com/teams/quarks/pipelines/cf-operator-nightly/jobs/clean-ns/builds/55#L5d1a1814:262 (as an example) @manno